src/descriptor/mod.rs

```diff
@@ -25,18 +25,22 @@
 
 use bitcoin::{self, Script, SigHashType};
 use bitcoin::blockdata::script;
-use bitcoin_hashes::sha256;
+use bitcoin_hashes::{sha256};
 use secp256k1;
 use std::fmt;
 use std::str::{self, FromStr};
 
 use expression;
 use miniscript::Miniscript;
 use policy::AbstractPolicy;
+use miniscript::evaluate::{Primitives, StackElement};
 use Error;
 use pubkey_size;
 use ToPublicKey;
+use secp256k1::VerifyOnly;
 
+
+pub mod create_descriptor;
```

Comment:
Rather than `pub mod create_descriptor` can you do
```
mod create_descriptor;
pub use create_descriptor::witness_stack;
```
so that `create_descriptor` is not exposed in the API as a separate module?